cypress/integration/visual-test/Pog-dark_spec.js

```diff
@@ -0,0 +1,15 @@
+describe('Pog visual dark mode regression check', () => {
```

Comment:
Should these files be Flow typed? Have we been add Flow annotations to the other visual test files?

Reply:
Have we been add Flow annotations to the other visual test files?
NO